scripts/cybershake/auto_submit.py

```diff
@@ -80,6 +80,7 @@ def submit_task(
             {"submit_time": submitted_time},
         )
 
+    # Merge ts
```

Comment:
Does it really need the comment if the first line checks if it is mergets?

Reply:
Haha probably not, but most of the others if have it.